src/components/Footer.js

```diff
@@ -55,41 +62,23 @@ const Footer = () => {
   return (
     <ContentWrapper>
       <FooterContainer>
-        <FooterFlexWrapperFirst>
-          <p
-            style={{
-              fontSize: "13px",
-              fontWeight: "bold",
-              letterSpacing: "0.1em",
-            }}
-          >
-            GET IN TOUCH
-          </p>
+        <LeftHalfOfFooterContainer>
+          <BoldText>GET IN TOUCH</BoldText>
           <p>
             <EmailLink href="mailto:Johnuxui@gmail.com">
               Johnuxui@gmail.com
             </EmailLink>
           </p>
-          <WithinFooterFirst
-            style={{
-              fontSize: "12px",
-            }}
-          >
+          <TextFontTypeOne>
```

Comment:
This naming is a little confusing bc you have an image in it. Can we come up with a better name?